### PR TITLE
Ensure discovery unsafe

### PR DESCRIPTION
**Pull request by bcoca**
Mark discovery data as 'unsafe' before using it

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

core

### CODE REVIEW

File: changelogs/fragments/ensure_discovery_unsafe.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - Wrap intepreter discovery result to prevent templating
```
Comment (abadger) [inline changelogs/fragments/ensure_discovery_unsafe.yml]: I think we do need to note that this is a security issue but if we say that it is only present in <strike>alpha1</strike> alpha1 and beta1, that should set expectations around how widely it affects people.

**UPDATED**: We decided that beta1 was okay to ship without this fix, so comment updated to say this is present in alpha1 and beta1.

Comment (bcoca) [reply changelogs/fragments/ensure_discovery_unsafe.yml]: do we have a `security:`  header?

Comment (abadger) [reply changelogs/fragments/ensure_discovery_unsafe.yml]: ```changelogs/config.yaml``` says no.  That's probably a heading we should add in the future.

**Review by jamescassell — APPROVED** [review]
shipit

**Review by nitzmahone — APPROVED** [review]

**Pull request merged**